docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+
+
+5. Note the cluster **Name** and **ID**.
+
+   > Swarm provisioning
+   >
+   > Your cluster is provisioned in two stages, and takes a few minutes to provision. Don't try to modify your cluster just yet!
+   > First, the manager node is deployed. Then, the additional infrastructure resources are deployed, including the worker nodes, DTR nodes, load balancers, subnet, and NFS volume.
+
+   * **Provisioning Stage 1**: Check the status of the manager node:
+   {% raw %}
+   ```bash
+   $ docker logs cluster-name_ID
+
+   Apply complete! Resources: 3 added, 0 changed, 0 destroyed.
+
+   The state of your infrastructure has been saved to the path
+   below. This state is required to modify and destroy your
+   infrastructure, so keep it safe. To inspect the complete state
+   use the `terraform show` command.
+
+   State path:
+
+   Outputs:
+
+   manager_public_ip = 169.##.###.##
+   swarm_d4ic_id = ID
+   swarm_name = cluster-name
+   ucp_password = UCP-password
+   ```
+   {% endraw %}
+
+   * **Provisioning Stage 2**: Check the status of the cluster infrastructure:
+    {% raw %}
+    ```bash
+     $ bx d4ic show --swarm-name cluster-name --sl-user user.name.1234567 --sl-api-key api_key
+     Getting swarm information...
+     Infrastructure Details
+
+     Swarm
+     ID           ID
+     Name         cluster-name
+     Created By   user.name.1234567
+
+     Nodes
+     ID         Name                           Public IP       Private IP      CPU   Memory   Datacenter   Infrakit Group
+     46506407   cluster-name-mgr1              169.##.###.##   10.###.###.##   2     4096     wdc07        managers
+     ...
+
+     Load Balancers
+     ID                                     Name                Address                                          Type
+     ID-string                              cluster-name-mgr    cluster-name-mgr-1234567-wdc07.lb.bluemix.net    mgr
+     ...
+
+     Subnets
+     ID        Gateway          Datacenter
+     ID-number 10.###.###.##   wdc07
+
+     NFS Volumes
+     ID              ID-number
+     Mount Address   fsf-wdc0701b-fz.adn.networklayer.com:/ID_number/data01
+     Datacenter      wdc07
+     Capacity        20
+     Type            ENDURANCE_FILE_STORAGE
+     Tier Level      10_IOPS_PER_GB
+
+     OK
+    ```
+    {% endraw %}
+
+After creating the cluster, you must [log in to Docker UCP and download the Docker UCP client certificate bundle](#use-the-universal-control-plane).
+
+## Use the Universal Control Plane
+Docker EE for IBM Cloud uses [Docker Universal Control Plane (UCP)](/datacenter/ucp/2.2/guides/) to provide integrated container management and security, from development to production.
+
+### Access UCP
+Before you begin, [create a cluster](#create-swarms). Note the UCP password.
+
+1. Retrieve your UCP password by using the cluster **Name** and **ID** that you made when you [created the cluster](#create-swarms).
+   ```bash
+   $ docker logs cluster-name_ID
```

Comment:
We prefer the notation that uses the object, so this would be `docker service logs cluster-name_ID`